consensus/istanbul/config.go

```diff
@@ -44,3 +48,10 @@ var DefaultConfig = &Config{
 	Epoch:          30000,
 	SubGroupSize:   21,
 }
+
+// Test block numbers used for the tests which uses hard fork block number information
+var (
+	TestBlockBeforeHF = big.NewInt(4)
+	TestBlockHF       = big.NewInt(5)
+	TestBlockAfterHF  = big.NewInt(6)
+)
```

Comment:
Do they need to be global variables? I think it can be declared on each test file.

Reply:
As you mentioned in another comment, the global config is accessed simultaneously by multiple test functions. I was trying to solve the problem with those variables :) It is removed with new change!

Reply:
They still exist for now, could you please check?